site.yml

```diff
@@ -8,7 +8,8 @@ content:
   sources:
   - url: https://github.com/owncloud/docs.git
     branches: 
-    - master
```

Comment:
This branch should only build `HEAD` and url `.`

Reply:
Why so? I get that HEAD would make sense, but `.`? Why would that value be required?

Reply:
Because it should only build the changes from the current branch.

Reply:
But as this will be merged into the 10.1 branch, it needs to reference the preceding branch, no? Or have I misunderstood you?